cli/src/pcluster/constants.py

```diff
@@ -108,6 +108,10 @@
 MIN_SLURM_NODE_PRIORITY = 1
 MAX_SLURM_NODE_PRIORITY = 2**32 - 1  # max value of uint32_t
 
+# Thresholds used to trigger a warning if Memory Based Scheduling and Flexible Instance Types are used together
+MIN_MEMORY_ABSOLUTE_DIFFERENCE = 2048
+MIN_MEMORY_PRECENTAGE_DIFFERENCE = 0.25
```

Comment:
Also, I would expect fairly large instances are more relevant for HPC, meaning that even a small relative difference can result in fairly high absolute differences. I would make this 20 %, maybe even less. To be discussed internally.